__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const testData = "../db/data/test-data";
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("Any url that does not have a route to the api", () => {
+  test("GET 404: responds with an error code and message when a wrong request is made", () => {
+    return request(app)
+      .get("/not-a-url")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Route not found");
+      });
+  });
+});
+
+describe("/api/topics", () => {
+  test("GET 200: responds with an array of topic objects with properties slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
```

Comment:
This is also nice destructuring. I love them both - but really they should be kept consistent so pick your favourite and then refactor the other one please